docs/config_search.md

```diff
@@ -276,6 +280,106 @@ profile_models:
 
 ---
 
+## Optuna Search Mode
+
+**-ALPHA RELEASE-**
+
+_This mode has the following limitations:_
+
+- **Ensemble, BLS or concurrent multi-model profiling is not supported**
+- **Profiling with request rate is not supported**
+
+This mode uses a hyperparameter optimization framework to search the configuration
+space, looking for the maximal objective value within the specified constraints.
+Please see the [Optuna](https://optuna.org/) website if you are interested in specific details on how the algorithm functions.
+
+Optuna allows you to search for every parameter that can be specified in the model configuration. Parameters can be specified
+with a min/max range (using the run-config-search options) or a list of parameters to test against can be set in the
+parameters/model_config_parameters field.
+
+After optuna search has found the best config(s), it will then sweep the top-N configurations found (specified by `--num-configs-per-model`) over the default concurrency range before generation of the summary reports.
+
+---
+
+_An example model analyzer YAML config that performs an Optuna Search:_
+
+```yaml
+model_repository: /path/to/model/repository/
+
+run_config_search_mode: optuna
+profile_models:
+  - model_A
+```
+
+A number of new configuration options were added to support tailoring the Optuna search to your needs:
```

Comment:
Looking at the actual rendered page, this is part of the section above, but it seems related to the thing below it, not the thing above it. I think the horizontal bars need to be tweaked

Reply:
I followed the same formatting as all the other sections. It looks fine to me when it's rendered.